frontend/src/components/MeetingRoom/SmallViewportHeader/SmallViewportHeader.tsx

```diff
@@ -33,7 +33,7 @@ const SmallViewportHeader = (): ReactElement => {
   return (
     <div
       className="flex items-center justify-between bg-darkGray-100 px-4 pt-4 text-white"
-      data-testId="smallViewportHeader"
+      data-testid="smallViewportHeader"
```

Comment:
nice one, I'm excited to not see those warnings anymore 😂 seems like something that lint should catch, we can look for it (not in this ticket though)